lib/commands/option-model.json

```diff
@@ -11,6 +11,12 @@
     "alias": null,
     "stringInput": "NONE"
   },
+  "force-deploy": {
+    "name": "force-deploy",
+    "description": "Forces ASK CLI deploy even if the hash of current skill package folder does not change.",
```

Comment:
To change the description to "Forces ASK CLI deploy skill package even if the hash of current skill package folder does not change." is more accurate?  Lambda code is always deployed.